test/node/src/delete.test.ts

```diff
@@ -244,6 +244,57 @@ for (const dialect of BUILT_IN_DIALECTS) {
 
         await query.execute()
       })
+
```

Comment:
let's add some test cases:

- `should delete from t1 returning *`
- `should delete from t1 using t2, t3 returning *`
- `should delete from t1 using t2, t3 returning t1.*, t2.*`

Reply:
Thanks for the suggestions! Added now.